Resources/Locale/en-US/_NF/ghost/roles/ghost-role-component.ftl

```diff
@@ -22,3 +22,7 @@ ghost-role-information-ert-mailcarrier-description = Assist with paperworks effo
 
 ghost-role-information-jerma-name = Jerma
 ghost-role-information-jerma-description = Pog moment
+
+ghost-role-information-baby-dragon-name = Space baby dragon
+ghost-role-information-baby-dragon-description = Hatch from your egg and go on incredible adventures with your mom and his crew!
+ghost-role-information-baby-dragon-rules = You are about to hatch from your egg, make sure to know who is your mom, be loyal and protect them!
```

Comment:
**Nitpick:** `You are about to hatch from your egg. Make sure to know who your mom is, be loyal and protect them!`

Avoids a run-on sentence, and uses correct syntax in the second clause. Also see above re mom's gender: use 'they' or 'she' consistently.

Reply:
resolved on last commit.